internal/components/setup/compose.go

```diff
@@ -67,7 +68,17 @@ func ComposeSetup(e2eConfig *config.E2EConfig) error {
 		return fmt.Errorf("bind wait ports error: %v", err)
 	}
 
-	execError := compose.WithCommand([]string{"up", "-d"}).Invoke()
+	// build command
+	cmd := make([]string, 0)
+	if e2eConfig.Setup.InitSystemEnvironment != "" {
+		profilePath := util.ResolveAbs(e2eConfig.Setup.InitSystemEnvironment)
+		cmd = append(cmd, "--env-file", profilePath)
+		util.ExportEnvVars(profilePath)
```

Comment:
You've already added it to the command option, why you need to export it ?

Reply:
It's goods to let `setup.steps` or `verify.query` commands could be using these environments too.